zillow_crawler/main.py

```diff
@@ -1,37 +1,51 @@
+from pathlib import Path
+
 from rich.columns import Columns
 from rich.console import Console
 from rich.live import Live
 from rich.panel import Panel
 from rich.table import Table
+from scrapy import signals
 from scrapy.crawler import CrawlerProcess
+from scrapy.statscollectors import StatsCollector
 from scrapy.utils.project import get_project_settings
 from sqlalchemy.orm import sessionmaker
 
-from zillow_crawler.models import Book, engine
-from zillow_crawler.spiders.books_toscrape import BooksToscrapeSpider
+from zillow_crawler.models import ZillowListing, ZillowPage, engine
+from zillow_crawler.spiders.zillow_houses import ZillowHousesSpider
+
+collected_stats = None
 
 
 def main():
     console = Console()
     process = CrawlerProcess(
         settings=get_project_settings(),
     )
-    process.crawl(BooksToscrapeSpider)
+    clawler = process.create_crawler(ZillowHousesSpider)
+    process.crawl(clawler)
     process.start()
 
+    stats = clawler.stats.get_stats()
+    console.print(stats)
+
     with Live(console=console, refresh_per_second=4) as live:
         cons = console
         session = sessionmaker(bind=engine)()
-        books = session.query(Book).all()
-
-        renderables = [
-            Panel(
-                f"[bold italic]{book.book_name}[/bold italic]\n[green]{book.product_price}[/green]",
-                border_style="yellow",
-            )
-            for book in books
-        ]
-        live.update(Panel(Columns(renderables, equal=True), title="Books"))
+        listings = session.query(ZillowListing).all()
+        pages = session.query(ZillowPage).all()
+
+        for listing in listings:
+            cons.print(listing.address)
+
```

Comment:
@OLmariana 
This is where your code should go for showing the listings and pages.